public/js/player-page.js

```diff
@@ -174,7 +174,7 @@ function loadTeam() {
 	} else {
 		// sets color
 		document.getElementById("player-header").style.backgroundColor = team.primaryColor;
```

Comment:
As an alternative (or addition) to the previous comment, you could set the style of the body to the primary color, and remove the background image entirely. it might make the area around the header look cleaner, and the rest of the page has manually defined colors in the css sheet.